### PR TITLE
Base updates of Bitcoin & LND

### PR DESCRIPTION
**Pull request by rootzoll**
including fresh python protobuffs for lnd 0.17.x

### CODE REVIEW

**Comment by rootzoll** [comment]
@openoms would like to merge this soon for an rc1 for the RPi5s

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## What's new in Version 1.11.0 of RaspiBlitz?
 
+- Update: Bitcoin Core v25.1 [details](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-25.1.md)
+- Update: LND v0.17.1-beta [details](https://github.com/lightningnetwork/lnd/releases/tag/v0.17.1-beta)
```
Comment (openoms) [inline CHANGES.md]:
```suggestion
- Update: LND v0.17.2-beta [details](https://github.com/lightningnetwork/lnd/releases/tag/v0.17.2-beta)
```

File: home.admin/config.scripts/lnd.install.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@
 ## based on https://raspibolt.github.io/raspibolt/raspibolt_40_lnd.html#lightning-lnd
 ## see LND releases: https://github.com/lightningnetwork/lnd/releases
 ### If you change here - make sure to also change interims version in lnd.update.sh #!
-lndVersion="0.16.4-beta"
+lndVersion="0.17.1-beta"
```
Comment (openoms) [inline home.admin/config.scripts/lnd.install.sh]: There has been  a quick bugfix release:
https://github.com/lightningnetwork/lnd/releases/tag/v0.17.2-beta
```suggestion
lndVersion="0.17.2-beta"
```

**Review by openoms — APPROVED** [review]
Ready to be merged to have the new automatic build ready for testing.

**Pull request merged**